src/fragments/lib/storage/js/list.mdx

```diff
@@ -95,3 +95,19 @@ Storage.list('photos/', { level: 'private' })
     .then(result => console.log(result))
     .catch(err => console.log(err));
 ```
+
+## Access all files 
+
+To get a list of all files in your S3 bucket under a specific prefix, you can set maxKeys to ALL.
```

Comment:
I would mention what is the default, what do you think?

Reply:
Yeah. that would totally makes sense.